demos/place_recognition_demo/python/place_recognition_demo/place_recognition.py

```diff
@@ -0,0 +1,96 @@
+"""
+ Copyright (c) 2021 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import numpy as np
+
+import cv2
+from tqdm import tqdm
+from pathlib import Path
+
+from place_recognition_demo.common import crop_resize, l2_distance
+
+from openvino.inference_engine import IECore # pylint: disable=no-name-in-module
+
+
+class IEModel(): # pylint: disable=too-few-public-methods
+    """ Class that allows working with Inference Engine model. """
+
+    def __init__(self, model_path, device, cpu_extension):
+        ie = IECore()
+        if cpu_extension and device == 'CPU':
+            ie.add_extension(cpu_extension, 'CPU')
+
+        path = '.'.join(model_path.split('.')[:-1])
```

Comment:
maybe with_suffix() method from pathlib.Path can be more human readable for these purpoces?

Reply:
Done